README.md

```diff
@@ -41,6 +42,10 @@ fastify
     })
   })
 ```
+
+This plugin support `callback` and `Promise` returned by the functions. Note that an `async` function
+doesn't have to use the `done` parameter.
```

Comment:
Can you include the docs for callbacks as well?

Reply:
I should have added a better description

Reply:
I mean, having both example is better.

Reply:
Done 👍